src/devices/Ft232H/Ft232HDevice.cs

```diff
@@ -0,0 +1,857 @@
+                case "ADBUS4":
+                case "D4":
+                case "GPIOL0":
+                    return 4;
+                case "ADBUS5":
+                case "D5":
+                case "GPIOL1":
+                    return 5;
+                case "ADBUS6":
+                case "D6":
+                case "GPIOL2":
+                    return 6;
+                case "ADBUS7":
+                case "D7":
+                case "GPIOL3":
+                    return 7;
+                case "ACBUS0":
+                case "C0":
+                case "GPIOH0":
+                    return 8;
+                case "ACBUS1":
+                case "C1":
+                case "GPIOH1":
+                    return 9;
+                case "ACBUS2":
+                case "C2":
+                case "GPIOH2":
+                    return 10;
+                case "ACBUS3":
+                case "C3":
+                case "GPIOH3":
+                    return 11;
+                case "ACBUS4":
+                case "C4":
+                case "GPIOH4":
+                    return 12;
+                case "ACBUS5":
+                case "C5":
+                case "GPIOH5":
+                    return 13;
+                case "ACBUS6":
+                case "C6":
+                case "GPIOH6":
+                    return 14;
+                case "ACBUS7":
+                case "C7":
+                case "GPIOH7":
+                    return 15;
+                default:
+                    return -1;
+            }
+        }
+
+        /// <summary>
+        /// Instantiates a FT232H device object.
+        /// </summary>
+        /// <param name="flags">Indicates device state.</param>
+        /// <param name="type">Indicates the device type.</param>
+        /// <param name="id">The Vendor ID and Product ID of the device.</param>
+        /// <param name="locId">The physical location identifier of the device.</param>
+        /// <param name="serialNumber">The device serial number.</param>
+        /// <param name="description">The device description.</param>
+        public Ft232HDevice(FtFlag flags, FtDeviceType type, uint id, uint locId, string serialNumber, string description)
+        : base(flags, type, id, locId, serialNumber, description)
+        {
+        }
+
+        /// <summary>
+        /// Instantiates a FT232H device object.
+        /// </summary>
+        /// <param name="ftDevice">a FT Device</param>
+        public Ft232HDevice(FtDevice ftDevice)
+            : base(ftDevice.Flags, ftDevice.Type, ftDevice.Id, ftDevice.LocId, ftDevice.SerialNumber, ftDevice.Description)
+        {
+        }
+
+        /// <summary>
+        /// Creates I2C bus related to this device
+        /// </summary>
+        /// <returns>I2cBus instance</returns>
+        /// <remarks>You can create either an I2C, either an SPI device.</remarks>
+        public I2cBus CreateI2cBus() => new Ft232HI2cBus(this);
```

Comment:
should this be abstract on the FtDevice? Same question for GPIO, SPI - we could return null or throw if given device doesn't support the protocol

Reply:
I was thinking of this. We can definitely do it for I2CBuc, SpiDevice and GpioDriver

Reply:
>  we could return null or throw if given device doesn't support the protocol

If it doesn't support it, I'd vote throw